README.md

```diff
@@ -45,118 +67,81 @@ The `beta to` and `beta from` commands can convert Ion to and from other formats
 Currently, JSON is supported.
 
 Convert Ion to JSON:
+
 ```shell
 ion beta to json my_file.10n
 ```
 
 Convert JSON to Ion:
+
 ```shell
 ion beta from json my_file.json
 ```
 
 ### Analyzing binary Ion file encodings with `inspect`
 
-The `beta inspect` command can display the hex bytes of a binary Ion file alongside
+The `inspect` command can display the hex bytes of a binary Ion file alongside
 the equivalent text Ion for easier analysis.
 
 ```shell
 # Write some text Ion to a file
 echo '{foo: null, bar: true, baz: [1, 2, 3]}' > my_file.ion
 
 # Convert the text Ion to binary Ion
-ion dump --format binary my_file.ion > my_file.10n
+ion cat --format binary my_file.ion > my_file.10n
 
 # Show the binary encoding alongside its equivalent text 
-ion beta inspect my_file.10n
-
----------------------------------------------------------------------------
- Offset   |  Length   |        Binary Ion        |         Text Ion
----------------------------------------------------------------------------
-          |         4 | e0 01 00 ea              |  // Ion 1.0 Version Marker
-        4 |         4 | ee 95 81 83              |  '$ion_symbol_table':: // $3::
-        8 |        19 | de 91                    |  {
-       10 |         1 | 86                       |    'imports': // $6:
-       11 |         2 | 71 03                    |    $ion_symbol_table, // $3
-       13 |         1 | 87                       |    'symbols': // $7:
-       14 |        13 | bc                       |    [
-       15 |         4 | 83 66 6f 6f              |       "foo",
-       19 |         4 | 83 62 61 72              |       "bar",
-       23 |         4 | 83 62 61 7a              |       "baz",
-          |           |                          |    ],
-          |           |                          |  }
-       27 |        13 | dc                       |  {
-       28 |         1 | 8a                       |    'foo': // $10:
-       29 |         1 | 0f                       |     null,
-       30 |         1 | 8b                       |    'bar': // $11:
-       31 |         1 | 11                       |     true,
-       32 |         1 | 8c                       |    'baz': // $12:
-       33 |         7 | b6                       |    [
-       34 |         2 | 21 01                    |       1,
-       36 |         2 | 21 02                    |       2,
-       38 |         2 | 21 03                    |       3,
-          |           |                          |    ],
-          |           |                          |  }
```

Comment:
🗺️ Now that `inspect` uses a variety of colors/styles, copy/pasted terminal output doesn't do it justice. I replaced each output example with a screenshot.